docker/lib/dependabot/docker/file_fetcher.rb

```diff
@@ -7,26 +7,44 @@ module Dependabot
   module Docker
     class FileFetcher < Dependabot::FileFetchers::Base
       def self.required_files_in?(filenames)
-        filenames.any? { |f| f.match?(/dockerfile/i) }
+        filenames.any? { |f| f.match?(/dockerfile/i) } or
+          filenames.any? { |f| f.match?(/^[^\.]+\.ya?ml$/i) }
```

Comment:
Bit pedantic, but, I notice we check for this regex in a few different places, sometimes with the `$` and sometimes not, would it make sense to extract it into a constant and reference that everywhere?

Reply:
done (and for `/dockerfile/i` also)